modules/statistics/php/stats_demographic.class.inc

```diff
@@ -227,6 +227,11 @@ class Stats_Demographic extends \NDB_Form
         $this->tpl_data['Subprojects'] = $subprojects;
         $this->tpl_data['Visits']      = $Visits;
 
+        // PREVENT Undefined property Console Output
+        if (!isset($this->params)) {
+            $this->params = null;
```

Comment:
Why not just define `params` as a class variable at the start of the class and instantiate it as null so that it's always defined?

Reply:
Way better solution, will do!